docs/adr/0006-ingestion-usage-metrics.md

```diff
@@ -0,0 +1,72 @@
+# Ingestion of Output Port Usage Metrics
+
+## Context and Problem Statement
+We must get daily aggregated usage data from producers (like Snowflake) into our application. 
+
+The ingestion mechanism must be simple for producers, who will need to run two different aggregation queries on their raw query logs and push the results to us.
+
+## Decision Drivers
+
+* Developer Experience: Must be simple for producers, even though they now have to provide two distinct metrics.
+* Reliability: The pipeline must be robust (retries, error handling).
+* Separation of Concerns: We must define a clear boundary of responsibility between DPP and the producer's systems.
+* Atomic Pushes: The push for both query stats and asset stats should be treated as a single unit of work for a given day.
+
+## Considered Options
+
+* **Option 1: Direct REST Endpoint Push** Producers use the REST endpoint directly in their pipeline. We provide the API endpoint and documentation. Producers are responsible for implementing all HTTP logic: authentication, request formatting, JSON serialization, and error/retry handling. They are also 100% responsible for generating the aggregation queries.
+* **Option 2: Fluent Python SDK (Simple API Wrapper)** We offer an SDK to give a more friendly interface to push this data. We provide a lightweight Python SDK (data-portal-sdk). This SDK handles all API complexity: authentication, request formatting, and retries. Producers are still responsible for running their own aggregation queries (we provide SQL snippets), but they simply pass the resulting list of dictionaries to a single SDK method.
```

Comment:
We can add to this, that we want to generate it from openapi spec in the end, but we can start with a simple single method